src/main/java/uk/gov/hmcts/reform/blobrouter/data/envelopes/Envelope.java

```diff
@@ -26,7 +26,7 @@ public Envelope(
         Status status,
         boolean isDeleted,
         boolean pendingNotification,
-        Integer fileSize
+        int fileSize
```

Comment:
I mean are you going to keep the size as Bytes?
If we process 4GB file int would not be enough.??